crates/router/src/core/webhooks/incoming.rs

```diff
@@ -1052,20 +1039,17 @@ async fn refunds_incoming_webhook_flow(
 
     // If event is NOT an UnsupportedEvent, trigger Outgoing Webhook
     if let Some(outgoing_event_type) = event_type {
-        let refund_response: api_models::refunds::RefundResponse =
-            updated_refund.clone().foreign_into();
-        Box::pin(super::create_event_and_trigger_outgoing_webhook(
+        Box::pin(super::add_bulk_outgoing_webhook_task_to_process_tracker(
             state,
-            merchant_context,
-            business_profile,
+            &business_profile,
+            &refund_id,
             outgoing_event_type,
-            enums::EventClass::Refunds,
-            refund_id,
-            enums::EventObjectType::RefundDetails,
-            api::OutgoingWebhookContent::RefundDetails(Box::new(refund_response)),
-            Some(updated_refund.created_at),
+            diesel_models::enums::EventClass::Refunds,
+            diesel_models::enums::EventObjectType::RefundDetails,
+            Some(refund.created_at),
         ))
-        .await?;
+        .await
+        .map_err(|e| e.change_context(errors::ApiErrorResponse::WebhookProcessingFailure))?;
```

Comment:
Same comments as before, about the `enums::` qualification and the unnecessary `map_err()`.